nvim/init.lua

```diff
@@ -1,18 +1,11 @@
+vim.loader.enable()
+
 local cmd = vim.cmd
-local fn = vim.fn
 local opt = vim.o
-local g = vim.g
 
 -- <leader> key. Defaults to `\`. Some people prefer space.
--- g.mapleader = ' '
--- g.maplocalleader = ' '
-
-opt.compatible = false
-
--- Enable true colour support
-if fn.has('termguicolors') then
-  opt.termguicolors = true
-end
+-- vim.g.mapleader = ' '
+-- vim.g.maplocalleader = ' '
```

Comment:
suggestion: add a comment explaining this.

```suggestion
-- The default leader is '\'. Some people prefer <space>. Uncomment this if you do, too.
-- vim.g.mapleader = ' '
-- vim.g.maplocalleader = ' '
```